### PR TITLE
Settle on notation: is_square(y), has_square_y(P)

### PR DESCRIPTION
**Pull request by sipa**
Solves #118

### CODE REVIEW

File: bip-taproot.mediawiki [hunk 1]
```diff
@@ -253,8 +253,7 @@ This function returns the witness stack necessary and a <code>sighash</code> fun
 def taproot_sign_script(internal_pubkey, script_tree, script_num, inputs):
     info, h = taproot_tree_helper(script_tree)
     (leaf_version, script), path = info[script_num]
-    _, is_y_quad = taproot_tweak_pubkey(internal_pubkey, t)
-    output_pubkey_tag = 0 if is_y_quad else 1
```
Comment (jonasnick) [inline bip-taproot.mediawiki]: was that line deleted accidentally?

File: bip-schnorr/test-vectors.py [hunk 2]
```diff
@@ -30,10 +30,10 @@ def vector2():
     msg = bytes_from_int(0x5E2D58D8B3BCDF1ABADEC7829054F90DDA9805AAB56C77333024B9D0A508B75C)
     sig = schnorr_sign(msg, seckey)
 
-    # This singature vector would not verify if the implementer checked the
+    # This signature vector would not verify if the implementer checked the
     # jacobi symbol of the X coordinate of R instead of the Y coordinate.
```
Comment (jonasnick) [inline bip-schnorr/test-vectors.py]: jacobi left over

Comment (sipa) [reply bip-schnorr/test-vectors.py]: Fixed.

File: bip-taproot.mediawiki (hunk 1 above)
Comment (sipa) [reply bip-taproot.mediawiki]: Yes, fixed.

**Comment by sipa** [comment]
@jonasnick Thanks, addressed your comments and removed the reference to "negative points".

**Comment by real-or-random** [comment]
ACK

**Comment by ajtowns** [comment]
ACK

**Pull request merged**